packages/cli/lib/services/response-saver.service.ts

```diff
@@ -153,9 +153,12 @@ function computeConfigIdentity(config: AxiosRequestConfig): ConfigIdentity {
 
     let headers: [string, string][] = [];
     if (config.headers !== undefined) {
-        const filteredHeaders = Object.entries(config.headers).filter(([key]) => !FILTER_HEADERS.includes(key.toLowerCase()));
+        const filteredHeaders = Object.entries(config.headers)
+            .filter(([key]) => !FILTER_HEADERS.includes(key.toLowerCase()))
+            .map<[string, string]>(([key, value]) => (key.toLowerCase().startsWith('nango-proxy-') ? [key.slice(12), String(value)] : [key, String(value)]));
         sortEntries(filteredHeaders);
         headers = filteredHeaders;
+        console.log(JSON.stringify(headers));
```

Comment:
Assume this is left over from debugging?

Reply:
Oops, yep. Fixed in 0e612b381